lib/circlemator/code_analyser.rb

```diff
@@ -2,27 +2,40 @@
 require 'httparty'
 require 'json'
 require 'pronto'
-require 'pronto/rubocop'
 require 'pronto/commentator'
 require 'circlemator/pr_finder'
 
 module Circlemator
-  class StyleChecker
+  class CodeAnalyser
     def initialize(opts)
       @opts = opts
       @base_branch = opts.fetch(:base_branch)
     end
 
-    def check!
+    def check_style
+      require 'pronto/rubocop'
+      run_pronto
+    end
```

Comment:
That doesn't look right 🤕

Reply:
Not sure where the problem is, because `pronto run` reports on the expected line number
```
circlemator mrgrodo$ pronto run -r undercover -c master
lib/circlemator/code_analyser.rb:20 W: instance method check_coverage missing tests for lines 21, 22 (coverage: 0.0)
```